jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -709,6 +710,79 @@ ecma_concat_ecma_strings (ecma_string_t *string1_p, /**< first ecma-string */
 } /* ecma_concat_ecma_strings */
 
 /**
+ * Format ecma-strings.
+ *
+ * @return a newly created ecma-string
+ */
+ecma_string_t *
+ecma_format_ecma_strings (const char * format, /**< format string */
+                          ...) /**< ecma-strings */
```

Comment:
I never particularly like `...`, for me it looks like a crude hack. I would prefer a constant array of ecma values. I am curious about others opinion.